presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimateMath.java

```diff
@@ -57,6 +57,7 @@ private static PlanNodeStatsEstimate differenceInStatsWithRangeStrategy(PlanNode
         return statsBuilder.setOutputRowCount(newRowCount).build();
     }
 
+    @Deprecated
```

Comment:
What should be used instead?

Reply:
We don't have a general purpose alternative. No surprise -- we don't know what this method should do, so hard to suggest a replacement. Perhaps `Random::new` ? ;)

Reply:
This method is kind of multiset subtract, but it is also used for negations (where it is not a good fit due to model limitations). Therefore one shouldn't use this method when not really needed.